src/Carter/DependencyContextAssemblyCatalog.cs

```diff
@@ -89,7 +89,9 @@ private static Assembly SafeLoadAssembly(AssemblyName assemblyName)
     {
         try
         {
+#pragma warning disable RS1035
```

Comment:
Should we add this to the NoWarn csproj list?

Reply:
No, since this warning would be valid if `DependencyContextAssemblyCatalog` were an analyzer. Since it is not, the warning can be suppressed. Just one small caveat of not splitting up library code and analyzers.